test/testCanonize.js

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright (c) 2024 Digital Bazaar, Inc. All rights reserved.
+ */
+import {base64UrlFromPdfBytes,
+  canonicalize,
+  hashCanonicalized} from '../lib/canonize.js';
```

Comment:
I can't speak for others, but I prefer the following style.  Either way, I think `index.js` can be used?
```suggestion
import {
  base64UrlFromPdfBytes,
  canonicalize,
  hashCanonicalized
} from '../lib/index.js';
```